octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 2;
+
```

Comment:
[Severity: Minor] The variable names 'a' and 'b' are not very descriptive.
Consider using more meaningful names like 'firstNumber' and 'secondNumber' to make the code clearer.
Rationale: Meaningful variable names enhance code readability and help others understand the purpose of the variables at a glance.
Reference: https://refactoring.guru/replace-parameter-with-method